### PR TITLE
Update Microsoft Support link for DualEngine API DMA requests.

### PR DESCRIPTION
**Pull request by AndrewLakeMSFT**
AB#51164048

### CODE REVIEW

**Review by captainbrosset — APPROVED** [review]

File: microsoft-edge/dualengine/reference/accelerator.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ api_location:
 
 > [!IMPORTANT]
 > The DualEngine API is part of a Limited Access Feature (see [LimitedAccessFeatures class](/uwp/api/windows.applicationmodel.limitedaccessfeatures)). For more information or 
-> to request an unlock token, contact [Microsoft Support](https://support.serviceshub.microsoft.com/supportforbusiness/create?sapId=d15d3aa2-0512-7cb8-1df9-86221f5cbfde).
+> to request an unlock token, contact [Microsoft Support](https://go.microsoft.com/fwlink/?linkid=2271232&clcid=0x409).
```
Comment (mikehoffms) [inline microsoft-edge/dualengine/reference/accelerator.md:28]:
```suggestion
> to request an unlock token, contact [Microsoft Support](https://go.microsoft.com/fwlink/?linkid=2271232).
```
Global in PR: remove &clcid=0x409 from this link
@captainbrosset 

Test:
https://go.microsoft.com/fwlink/?linkid=2271232&clcid=0x409
https://go.microsoft.com/fwlink/?linkid=2271232

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Writer/Editor review: remove & part of link

**Review by mikehoffms — APPROVED** [review]
Approved PR per Writer/Editor review.

File: microsoft-edge/dualengine/reference/accelerator.md (hunk 1 above)
Comment (captainbrosset) [reply microsoft-edge/dualengine/reference/accelerator.md:28]: This is a timely request so I'll merge as is, and open a new issue tro track removing that part of the URL.

**Pull request merged**